Makefile

```
@@ -27,34 +53,53 @@ tmp/packer_$(PACKER_VERSION).zip:
 	curl -fsSLo $@ 'https://releases.hashicorp.com/packer/$(PACKER_VERSION)/packer_$(PACKER_VERSION)_$(UNAME)_amd64.zip'
 	@echo Downloaded new Packer version: $(PACKER_VERSION)!
 
-validate: $(PACKER) ~/bin/packer-post-processor-vagrant-cloud-standalone
+# ~/bin/packer-post-processor-vagrant-cloud-standalone
+
+validate: $(PACKER)
 	$(PACKER) validate st2.json
-	$(PACKER) validate st2_deploy.json
 
-build: $(PACKER)
-	$(PACKER) build \
-	  -var 'st2_version=$(ST2_VERSION)' \
-	  -var 'box_version=$(BOX_VERSION)' \
-	  st2.json
+INSPEC = $(shell command -v inspec 2>/dev/null)
+install-inspec: # https://docs.chef.io/inspec/install
+ifeq (,$(INSPEC))
+	@{ \
+		case $(UNAME) in \
+			darwin) \
+				brew install chef/chef/inspec; \
+				;; \
+			linux) \
+				curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec; \
+				;; \
+		esac \
+	}
+	INSPEC = $(shell command -v inspec 2>/dev/null)
+endif
 
-# 'Vagrant-cloud-standalone' is a forked Packer post-processor plugin to deploy .box artifact to Vagrant Cloud
-install-vagrant-cloud-standalone: tmp/vagrant-cloud-standalone_$(VAGRANT_CLOUD_STANDALONE_VERSION).zip
-	mkdir -p ~/bin
-	unzip -o -d ~/bin $<
-	@echo Vagrant-cloud-standalone plugin $(VAGRANT_CLOUD_STANDALONE_VERSION) was successfully installed!
-tmp/vagrant-cloud-standalone_$(VAGRANT_CLOUD_STANDALONE_VERSION).zip:
-	curl -fsSLo $@ 'https://github.com/armab/packer-post-processor-vagrant-cloud-standalone/releases/download/v$(VAGRANT_CLOUD_STANDALONE_VERSION)/packer-post-processor-vagrant-cloud-standalone_$(UNAME)_amd64.zip'
-	@echo Downloaded new Vagrant-cloud-standalone version: $(VAGRANT_CLOUD_STANDALONE_VERSION)!
-# Install 'packer-post-processor-vagrant-cloud-standalone' only if it doesn't exist
-~/bin/packer-post-processor-vagrant-cloud-standalone:
-	@$(MAKE) install-vagrant-cloud-standalone
-
-# Deploy the .box, produced during the `build` to Vagrant Cloud
-deploy: $(PACKER) ~/bin/packer-post-processor-vagrant-cloud-standalone
+inspec-lint: $(INSPEC)
+	@{ \
+		cd test/integration/ && \
+		for dir in */; do \
+			dir=$$(basename $$dir) ; \
+			if [ -f "$${dir}/inspec.yml" ]; then \
+				echo -e "\nRunning Inspec lint for \033[1;36m$${dir}\033[0m ..."; \
+				inspec check --chef-license=accept-silent --diagnose $${dir}; \
+			fi \
+		done \
+	}
+
+build: $(PACKER) validate
 	$(PACKER) build \
-	  -var 'st2_version=$(ST2_VERSION)' \
-	  -var 'box_version=$(BOX_VERSION)' \
-	  st2_deploy.json
+		-var 'st2_version=$(ST2_VERSION)' \
+		-var 'box_version=$(BOX_VERSION)' \
+		-var 'box_org=$(BOX_ORG)' \
+		st2.json
+
+publish: $(VAGRANT)
+	vagrant cloud publish \
+		--description "Box with StackStorm (aka 'IFTTT for Ops') event-driven automation platform: auto-remediation, security responses, facilitated troubleshooting, complex deployments, ChatOps and more. \n* https://stackstorm.com/ \n* Documentation: https://docs.stackstorm.com/ \n* Community: https://stackstorm.com/community-signup \n* Forum: https://forum.stackstorm.com/" \
```

```suggestion
		--description "Box with StackStorm (aka 'IFTTT for Ops') event-driven automation platform: auto-remediation, security responses, facilitated troubleshooting, complex deployments, ChatOps and more. \n* https://stackstorm.com/ \n* Documentation: https://docs.stackstorm.com/ \n* Community: https://stackstorm.com/community-signup \n* Discussions: https://github.com/StackStorm/st2/discussions" \
```
The forum is read only. Now we use GitHub discussions.